### DISCUSSION TITLE
Settings icons

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#130] [Polls]
Icons can increase the size of patches.

Edit: Some people think that I made them just for design, for a better look. But my first thought was to be able to find the necessary options faster, because people identify visual objects better, and reading a text always takes longer.

I find some icons inconsistent that I'm willing to fix eventually. I invite other designers who want to fix them or add more icons.

**Comment by tooruu** [comment]
Incredibly useful feature!

**Reply by fuengfah** [reply]
How to use Youtube Original icon?

What to put in value in option.toml

**Reply by MtndaleRedneck** [reply]
> How to use Youtube Original icon?
> 
> What to put in value in option.toml

Just don't enable this patch.
